modules/rosa-kubernetes-concept.adoc

```diff
@@ -7,7 +7,7 @@
 [id="rosa-kubernetes-concept_{context}"]
 = Kubernetes
 
-{product-title} (ROSA) uses the Red Hat enterprise Kubernetes platform. Kubernetes is an open source platform for managing containerized workloads and services across multiple hosts, and offers management tools for deploying, automating, monitoring, and scaling containerized apps with minimal to no manual intervention. For complete information about Kubernetes, see the link:https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational[Kubernetes documentation].
+{product-title} (ROSA) uses Red Hat OpenShift, which is an enterprise Kubernetes platform. Kubernetes is an open source platform for managing containerized workloads and services across multiple hosts, and offers management tools for deploying, automating, monitoring, and scaling containerized apps with minimal to no manual intervention. For complete information about Kubernetes, see the link:https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational[Kubernetes documentation].
```

Comment:
Should Red Hat OpenShift be Red Hat OpenShift Container Platform? Could you verify?

Reply:
@sagidlow, confirmed that it's Red Hat Openshift. Here's the technology primer for Red Hat Cloud Services: https://docs.google.com/document/d/1uYYL4tv1_yCXIM8mK44ndxlIciBjbh9UTToKFBc3Ako/edit#heading=h.4giugr7ndee1

Reply:
@sayjadha Perfect! Thank you!